bedrock/mozorg/helpers/misc.py

```diff
@@ -349,11 +349,11 @@ def press_blog_url(ctx):
 @jingo.register.function
 @jinja2.contextfunction
 def donate_url(ctx, source=''):
-    """Output a link to the donation page taking locales into account.
+    """Output a link to the donation page formatted using settings.DONATE
 
-    Uses the locale from the current request. Checks to see if we have
-    a donation page that match this locale, returns the localized page
-    url or falls back to the US page url if not.
+    Formats the donation url using custom values defined in settings.DONATE for
+    the curent locale. If a locale is not defined, it falls back to default, except
+    for the locale, which is always the locale from the request.
```

Comment:
Typo: current. Kind of hard to follow the second paragraph (do you really need the part after except)